### DISCUSSION TITLE
Cannot find trait `FromRequest` in this scope

### DISCUSSION THREAD

**Discussion by CrayfishGo** [tokio-rs/axum#2502] [Q&A]
### Summary

![FB3E8943-6776-4c4c-9C51-2FD8897245C2](https://github.com/tokio-rs/axum/assets/62917071/45927373-1957-4875-8d15-7fb0ce60da53)

Why this error happened? How to fix it?

### axum version

0.7.2

**Comment by davidpdrsn** [comment]
Seems to be an issue with your IDE. Does `cargo check` work when you run it in a terminal?

**Reply by CrayfishGo** [reply]
`cargo check` works fine。 It seems to be an issue with the Rust plugin for IDEA。
